### PR TITLE
Add GHA build

### PR DESCRIPTION
**Pull request by gliptak**
Signed-off-by: Gábor Lipták <gliptak@gmail.com>

## Description

Travis no loner reports on PRs Replacing with GHA

https://github.com/gliptak/vimium/actions/runs/1485895315

Please review the "Which pull requests get merged?" section in `CONTRIBUTING.md`.

### CODE REVIEW

**Comment by gliptak** [comment]
@philc please review

**Review by planeiii-te — CHANGES_REQUESTED** [review]
A few small things to clean up.

**Comment by gliptak** [comment]
@planeiii-te updated

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+      run: |
+        npm install util puppeteer
+        ./make.js test
+    - name: Build
+      run: npx web-ext build --overwrite-dest
+    - name: Upload document
+      uses: actions/upload-artifact@v2
+      with:
+        name: vimium
+        path: web-ext-artifacts/
```
Comment (planeiii-te) [inline .github/workflows/ci.yml:29]: Minor: The indentation is off here, but it looks good otherwise.

**Review by planeiii-te — APPROVED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (gliptak) [reply .github/workflows/ci.yml]: updated

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  ci:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Setup Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: 10
+      - name: Test
```
Comment (fearphage) [inline .github/workflows/ci.yml:19]: Just noticed you're missing the actual `npm install`.

**Review by fearphage — APPROVED** [review]
Good to go after you add `npm install`.

File: .github/workflows/ci.yml (hunk 2 above)
Comment (gliptak) [reply .github/workflows/ci.yml:19]: @fearphage `setup-node` already includes `npm` you can review the build log at https://github.com/gliptak/vimium/runs/4276312033?check_suite_focus=true

**Comment by gliptak** [comment]
@fearphage setup-node already includes npm you can review the build log at https://github.com/gliptak/vimium/runs/4276312033?check_suite_focus=true

**Comment by gliptak** [comment]
@fearphage anything else for this PR?

**Review by fearphage — APPROVED** [review]
:shipit:

**Comment by gliptak** [comment]
who is to click "Merge"?

**Pull request closed without merging**